server.py

```diff
@@ -221,20 +222,6 @@ async def get_root(request):
         def get_embeddings(self):
             embeddings = folder_paths.get_filename_list("embeddings")
             return web.json_response(list(map(lambda a: os.path.splitext(a)[0], embeddings)))
-        
-        @routes.get("/models")
```

Comment:
We need to keep these routes for now if we want to merge this PR first. If we decide to land this PR after frontend changes has landed, the frontend side needs to be able to handle both APIs.

Reply:
These two APIs have not been removed, just moved to `model_filemanager/model_filemanager.py`. I can restore it if needed.

Reply:
The new API adds the prefix of `/experiment`. My plan is to retain the original API and then migrate it to the original API after the function is stable.